src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -17,11 +18,34 @@ import type { StakingId } from 'state/slices/opportunitiesSlice/types'
 import {
   selectAggregatedEarnUserLpOpportunity,
   selectAggregatedEarnUserStakingOpportunitiesIncludeEmpty,
+  selectFeatureFlags,
 } from 'state/slices/selectors'
 import { useAppSelector } from 'state/store'
 
 import { StakingTable } from './StakingTable'
 
+const exampleSaverVault: EarnOpportunityType = {
+  contractAddress: '0xa354f35829ae975e850e23e9615b11da1b3dc4de',
```

Comment:
Note we'll probably have to remove/revisit this type now that we'll be dealing with native assets. `EarnOpportunityType` is basically a legacy from the previous earn types which doesn't really fit what we represent anymore